Not providing labels for all classes leads to crash

Related to #127.

Add three classes but only labels for two and predict.
I got a shape mismatch as follows:
AssertionError:  ERROR: Operator <lazyflow.operators.classifierOperators.OpPredictRandomForest object at 0x444e490> has failed to provide a result of correct shape. result shape is (1, 256, 256, 1, 2) vs (1, 256, 256, 1, 3).  roi was "Subregion: start '[0, 256, 512, 23, 0]' stop '[1, 512, 768, 24, 3]'"

The random forest returned two channels but ilastik expected three.

--- a/ilastik/applets/pixelClassification/opPixelClassification.py
+++ b/ilastik/applets/pixelClassification/opPixelClassification.py
@@ -90,6 +90,7 @@ class OpPixelClassification( Operator ):
         self.opTrain = OpTrainRandomForestBlocked( parent=self, graph=self.graph )
         self.opTrain.inputs['Labels'].connect( self.opLabelPipeline.Output )
         self.opTrain.inputs['Images'].connect( self.CachedFeatureImages )
+        self.opTrain.inputs['MaxLabel'].connect( self.opMaxLabel.Output )
         self.opTrain.inputs["nonzeroLabelBlocks"].connect( self.opLabelPipeline.nonzeroBlocks )
         self.opTrain.inputs['fixClassifier'].setValue( False )
 

